cvmfs/options.h

```diff
@@ -55,7 +55,8 @@ class OptionsManager {
     }
   }
 
-  OptionsManager(const OptionsManager& opt_mgr) {
+  OptionsManager(const OptionsManager& opt_mgr)
+    : taint_environment_(true) {
```

Comment:
What I meant is: wouldn't it need to be

    taint_environment_ = opt_mgr.taint_environment_

?

Reply:
Yes, absolutely. Sorry about that - just made a push that fixes this.